calculator.js

```diff
@@ -3,3 +3,51 @@ const exampleAdditionInput = {
   num2: 5,
   operation: 'add',
 }
+
+const calculate = function (inputHash) {
+  leftOperator = inputHash.num1;
+  rightOperator = inputHash.num2;
+  operand = inputHash.operation;
```

Comment:
You forgot to use `const` here.  

Unfortunately even inside a function JavaScript creates a global variable if you don't use `let` or `const`.  (Yes, that _is_ as bad a decision as it seems like.  You aren't missing something.  JavaScript is shockingly good _for being designed in less than two weeks_, it's less good when you aren't grading on a curve.)

```suggestion
  const leftOperator = inputHash.num1;
  const rightOperator = inputHash.num2;
  const operand = inputHash.operation;
```